src/js/react/apps/linkedevents/components/ResultCard.tsx

```diff
@@ -69,6 +69,12 @@ function ResultCard({ end_time, id, location, name, keywords=[], start_time, ima
     return locationString;
   };
 
+  const getOffers = (): boolean => {
+    const offersInfoUrls = offers?.map(offer => offer.info_url[currentLanguage]);
+
+    return !!offersInfoUrls;
```

Comment:
I don't think `getOffers` works correctly. We discussed this in slack with more detail.

Reply:
I changed the functionality a bit now so that it checks the url validity or if its null or undefined. I tested it with few events and it seems to work correctly for me now 🙏